cinema/serializers.py

```diff
@@ -125,10 +125,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
+            ValidationError,
         )
```

Comment:
Ensure that the `validate_ticket` method in the `Ticket` model is correctly implemented and accessible. This method is used to validate the ticket's row, seat, and cinema hall, and if it's not defined, it will cause validation errors.